lesson_5_build/ex2.md

```diff
@@ -11,3 +11,6 @@ Edit the Dockerfile and/or write below your thoughts. Open a pull request with y
 
 ### Your changes
```

Comment:
Poderîas ter engadido máis cousas:
- cambiar a imaxe a lago máis minimalista (p.e. alpine), incluso unha imaxe que corra o binario e punto
- Facer un multistage para librarte do lixo que xera APT
- APT sen cache
- Borrar o zip